### DISCUSSION TITLE
tgchannels/

### DISCUSSION THREAD

**Discussion by giscus** [rusosnet/rusosnet.github.io#9] [General]
# tgchannels/

https://rusos.net/tgchannels/

**Comment by alexesDev** [comment]
https://t.me/+gLPDMeuX93s5NWM6 Личный канал итшницы
https://t.me/bocetosU Наброски о жизни искусствоведа в Уругвае и культурный взгляд на Монтевидео: куда сходить, что посмотреть и как тут дышится

**Reply by alexesDev** [reply]
Перенесено
